Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx

```diff
@@ -142,6 +148,43 @@ class StatusTaskModule extends React.Component<RouteComponentProps, IStatusState
             );
         }
     }
+
+    private audienceSelection = () => {
+        if (this.state.message.teamNames) {
+            let length = this.state.message.teamNames.length;
+            return (
+                <div>
+                    <h3>Sent to General channel in teams</h3>
+                    {this.state.message.teamNames.sort().map((team, index) => {
+                        if (length === index + 1) {
+                            return (<span key={`teamName${index}`} >{team}</span>);
+                        } else {
+                            return (<span key={`teamName${index}`} >{team}, </span>);
+                        }
+                    })}
+                </div>);
+        } else if (this.state.message.rosterNames) {
+            let length = this.state.message.rosterNames.length;
+            return (
+                <div>
+                    <h3>Sent to General channel in teams</h3>
```

Comment:
Change string to "Sent in chat to people in teams"

Reply:
done